terraform/eks_adot_operator_cluster_setup/main.tf

```diff
@@ -20,6 +20,12 @@ terraform {
       version = ">= 1.7.0"
     }
   }
+  ## comment out the backend block if you want the terraform state file to be locally saved on the computer (or) change the name of the s3 bucket if you are testing out on your personal AWS account
+  backend "s3" {
```

Comment:
The backend s3 module is part of the main terraform release. https://github.com/hashicorp/terraform/blob/main/website/docs/language/settings/backends/s3.mdx. So we can lock down the terraform version. But I am not sure if we want to do that as the tf script before the backend upgrade didn't lock down to a specific version. Lmk if its okay to do that

Reply:
I think it should be safe for this version but it may make sense to open an issue to see if it should be done as a whole for all modules in this repository.

Reply:
okay, pushed that into the new commit

Reply:
@bryan-aguilar is it okay we push this PR up? As per anuraag suggestion, I can lock the versions of others tf scripts on another PR